libraries/arrow-convert/src/lib.rs

```diff
@@ -27,3 +28,32 @@ impl DerefMut for ArrowData {
         &mut self.0
     }
 }
+
+macro_rules! register_array_handlers {
+    ($(($variant:path, $array_type:ty, $type_name:expr)),* $(,)?) => {
+        pub fn cast_values(data: &ArrowData) -> Result<Vec<f64>> {
```

Comment:
Sorry actually do you mind renaming this `into_vec_f64` for readibility:

```suggestion
        pub fn into_vec_f64(data: &ArrowData) -> Result<Vec<f64>> {
```